privacy_evaluator/models/tf/cnn.py

```diff
@@ -29,8 +49,23 @@ def ConvNet(
 
 
 class ConvNetMNIST(keras.Model):
+    """
+    Provide a convolutional neural network for MNIST classification.
+
+    Note: This method is just aimed at fetching a model for developers' test when
+    a target model is required. Since only the `MNIST` dataset is our concern, this 
+    method is compatible only with the corresponding image size (28*28).
+
+    Args:
+        num_classes: Number of classes during prediction, serving as the size of
+        the last fully-connected layer.
+        input_shape: Either (28, 28) or its variations (because of possible positions \
```

Comment:
Could you explain what you mean here by variation? The only variation of 28,28 I see would be 28,28. Maybe am I overlooking something? Otherwise, I would suggest leaving out the comment on the variations.